src/transi/transi.h

```diff
@@ -166,6 +167,8 @@ int trans_use_mpi(_bool);
  */
 int trans_init(void);
 
+int trans_set_mpi_comm(const MPI_Fint mpi_user_comm);
```

Comment:
Please change this to use a standard int as to not `#include  <mpi.h>`
```suggestion
int trans_set_mpi_comm(int);
```